app/models/consultation.rb

```diff
@@ -18,6 +18,7 @@ class AddNeighbourAddressesError < StandardError; end
   attribute :consultee_message_subject, :string, default: -> { default_consultee_message_subject }
   attribute :consultee_message_body, :string, default: -> { default_consultee_message_body }
 
+  attribute :deadline, :string
```

Comment:
Firstly - is this ever set? In the controller we call `update!` but the params are filtered to remove `:deadline`. Then in `extend_deadline` we deal with the param from the hash directly and not from the model.

I think it'd be better to call the attribute something like `deadline_extension` and define it as an integer. We can then leave it in the assigned params and then use a `before_save` callback to set the new end date if the value is present and greater than one.